matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/Event.kt

```diff
@@ -443,7 +447,7 @@ fun Event.isInvitation(): Boolean = type == EventType.STATE_ROOM_MEMBER &&
         content?.toModel<RoomMemberContent>()?.membership == Membership.INVITE
 
 fun Event.getPollContent(): MessagePollContent? {
-    return content.toModel<MessagePollContent>()
+    return getDecryptedContent().toModel<MessagePollContent>()
```

Comment:
It seems to not work well in non encrypted room. When testing in a non encrypted room, I could not see the poll question in the `replyTo` message.

Reply:
We should use `getClearContent()` instead: fixed in [4a65e11](https://github.com/vector-im/element-android/pull/7566/commits/4a65e1153abcd25a8927340cc3f20c396c9d2ef1)